versions/3.0.md

```diff
@@ -1820,7 +1820,7 @@ The key value used to identify the callback object is an expression, evaluated a
 ##### Patterned Fields
 Field Pattern | Type | Description
 ---|:---:|---
-<a name="callbackExpression"></a>{expression} | [Path Item Object](#pathItemObject) | A Path Item Object used to define a callback request and expected responses.  A [complete example](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/examples/v3.0/callback-example.yaml) is available.
+<a name="callbackExpression"></a>{expression} | [Path Item Object](#pathItemObject) | A Path Item Object used to define a callback request and expected responses.  A [complete example](/examples/v3.0/callback-example.yaml) is available.
```

Comment:
Hmm, I looked at it in the [differences view](https://github.com/OAI/OpenAPI-Specification/pull/1223/files?short_path=4a97aef#diff-4a97aef5bba9ddb7fd8609f48f4bdc30), and there it pointed to https://github.com/OAI/OpenAPI-Specification/blob/master/examples/v3.0/callback-example.yaml. Strangely, if I open the same address again in a new browser tab, it now points to https://github.com/OAI/OpenAPI-Specification/blob/rel_link/examples/v3.0/callback-example.yaml.  (Both links give 404s currently, the second one uses your branch name with the main fork's repository name.)

I guess Github it not that predictable with the link expansion.